src/official-account/official-account.ts

```diff
@@ -339,23 +374,23 @@ class OfficialAccount extends EventEmitter {
       // wechaty load the SelfContact object, so just return it.
       /* eslint-disable sort-keys */
       const selfContactPayload: OAContactPayload = {
-        subscribe         : 0,
-        openid            : openId,
-        nickname          : 'from official-account options ?',
-        sex               : ContactGender.Unknown,
-        language          : 'zh_CN',
-        city              : '北京',
-        province          : '北京',
-        country           : '中国',
-        headimgurl        : '',
-        subscribe_time    : 0,
-        unionid           : '0',
-        remark            : '微信公众号客服',
-        groupid           : 0,
-        tagid_list        : [],
-        subscribe_scene   : '',
-        qr_scene          : 0,
-        qr_scene_str      : '',
+        subscribe: 0,
+        openid: openId,
+        nickname: 'from official-account options ?',
+        sex: ContactGender.Unknown,
+        language: 'zh_CN',
+        city: '北京',
+        province: '北京',
+        country: '中国',
+        headimgurl: '',
+        subscribe_time: 0,
+        unionid: '0',
+        remark: '微信公众号客服',
+        groupid: 0,
+        tagid_list: [],
+        subscribe_scene: '',
+        qr_scene: 0,
+        qr_scene_str: '',
```

Comment:
Could we keep the spaces for better readability? Because I feel that each import should be a separate block visually.